modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc

```diff
@@ -23,14 +25,18 @@
 To be a user of the `cluster-admins` group, use the `oc adm groups new cluster-admins <user>` command, where `<user>` is the default cluster role that you can bind to users and groups cluster-wide or locally. 
 ====
 . To log in with your username and password, obtain the password for the Argo CD instance:
-.. In the left panel of the console, use the perspective switcher to switch to the *Developer* perspective.
-.. Use the *Project* drop-down list and select the `openshift-gitops` project.
-.. Use the left navigation panel to navigate to the *Secrets* page.
-.. Select the *openshift-gitops-cluster* instance to display the password.
+.. In the left panel of the console, use the perspective switcher to switch to the *Administrator* perspective.
```

Comment:
Removing this step as per your suggestion since it is duplicitous as we have already instructed the user to be in the Administrator perspective earlier.